daemon/algod/api/server/lib/middlewares/auth.go

```diff
@@ -25,7 +25,13 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// URLAuthPrefix is the echo formatted url/path param which can be used for supplying an API token.
+const URLAuthPrefix = "/urlAuth/:" + TokenPathParam
 const urlAuthFormatter = "/urlAuth/%s"
+// InvalidTokenMessage is the message set when an invalid / missing token is found.
+const InvalidTokenMessage = "Invalid API Token"
+// TokenPathParam is the name of the path parameter used by URLAuthPrefix
+const TokenPathParam = "token"
```

Comment:
It took me a moment to realize this works because the Go compiler is smart; I was expecting to need this declared before it's used a few lines above.

Reply:
It's been a while since forward declaration spent much time interrupting my train of thought. I'll go ahead and move this up anyway